lms/djangoapps/branding/toggles.py

```diff
@@ -8,8 +8,8 @@
 
 def use_catalog_mfe():
     """
-    Determine if Catalog MFE is enabled, replacing student_dashboard
+    Returns a boolean = true if the Catalog MFE is enabled.
     """
     return configuration_helpers.get_value(
-        'ENABLE_CATALOG_MICROFRONTEND', settings.FEATURES['ENABLE_CATALOG_MICROFRONTEND']
+        'ENABLE_CATALOG_MICROFRONTEND', settings.FEATURES.get('ENABLE_CATALOG_MICROFRONTEND')
     )
```

Comment:
You are now saying that this returns a boolean, which means in should never return `None` which can currently happen if the setting is not set.  Either set a default for this value here or in the common.py settings file (I would prefer in the common.py settings file).

Also note, we have [stopped using the `FEATURES` dictionary](https://github.com/openedx/edx-platform/issues/36890) as a part of the platform settings simplification work.  You should not use it in your new code and update anything that you're touching to drop its usage as it makes sense.  You don't need to fix everything but do the minimal necessary to not add new code that uses this deprecated setting.